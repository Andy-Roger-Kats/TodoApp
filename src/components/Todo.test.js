import React from "react";
import renderer from "react-test-renderer";
import Todo from "./Todo";

const TODO = [{ id: "todo-0", name: "Eat", completed: true }];

describe("The <Todo/> component", () => {
  it("should match its snapshot", () => {
    const tree = renderer
      .create(
        <Todo
          id={TODO[0].id}
          name={TODO[0].name}
          completed={TODO[0].completed}
        />
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
