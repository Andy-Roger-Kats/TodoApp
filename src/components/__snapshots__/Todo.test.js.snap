// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The <Todo/> component should match its snapshot 1`] = `
<li
  className="todo"
>
  <div
    className="stack-small"
  >
    <div
      className="c-cb"
    >
      <input
        defaultChecked={true}
        id="todo-0"
        onChange={[Function]}
        type="checkbox"
      />
      <label
        className="todo-label"
        htmlFor="todo-0"
      >
        Eat
      </label>
    </div>
    <div
      className="btn-group"
    >
      <button
        className="btn"
        onClick={[Function]}
        type="button"
      >
        Edit 
        <span
          className="visually-hidden"
        >
          Eat
        </span>
      </button>
      <button
        className="btn btn__danger"
        onClick={[Function]}
        type="button"
      >
        Delete 
        <span
          className="visually-hidden"
        >
          Eat
        </span>
      </button>
    </div>
  </div>
</li>
`;
